modules/assim.batch/R/pda.utils.R

```diff
@@ -668,9 +668,17 @@ pda.generate.sf <- function(n.knot, sf, prior.list){
 
 
 ##' @title return.bias
+##' @param settings settings list
+##' @param isbias bias variable index
+##' @param model.out model output list
+##' @param inputs inputs list
+##' @param prior.list.bias prior list, bias prior to be added
+##' @param run.round extension flag
+##' @param pass2bias if this is another round, this is re-sampled MCMC samples, will go with the rest od params
```

Comment:
what are "rest od params"?

Reply:
typo, should've been *of..meant "rest of model parameters" 

if next round's bias parameters are sampled from the previous round's MCMC together with the rest of the parameters, instead of using the fitted values, emulator resolves this parameter better